README.md

```diff
@@ -2463,6 +2473,25 @@ This is a route-map for filtering networks sent to this area. Valid values are s
 ##### `filter_list_out`
 This is a route-map for filtering networks sent from this area. Valid values are string or keyword 'default'.
 
+##### `nssa`
+This property defines the area as NSSA (not so stubby area). Valid values are true, false or keyword 'default'. This property is mutually exclusive with `stub` and `stub_no_summary`.
+
+##### `nssa_default_originate`
+This property configures to originate Type-7 default LSA into NSSA area. Valid values are true, false or keyword 'default'.

```

Comment:
`Generates an NSSA External (type 7) LSA for use as a default route to the external autonomous system.`